File: js/linearSearch.js
const arrayToSearchThrough = [];
for (let i = 1; i <= 1000; i++) {
    arrayToSearchThrough.push(i);
}

exports.linearSearch = function(valueToFind, arrayToSearchThrough) {
    for(i = 0; i < arrayToSearchThrough.length; i++) {
        if (arrayToSearchThrough[i] == valueToFind) {
            return i
        }
    }
};

exports.linearSearchGlobal = function(valueToFind, arrayToSearchThrough) {
    var results = []
    for(i = 0; i < arrayToSearchThrough.length; i++) {
        if (arrayToSearchThrough[i] == valueToFind) {
            results.push(i)
        }
    }
    return results
};

File: python/linear_search.py
array_to_search_through = []
for number in range(1, 1001):
    array_to_search_through.append(number)

def linear_search(value_to_find, array_to_search_through):
    # your code here
    # print('Value:',value_to_find)
    # print('Array:',array_to_search_through)

    new_index = []

    # iterating the length of list to search for value, return index of value
    for index in range(len(array_to_search_through)):
        if (array_to_search_through[index] == value_to_find):
            # print('Index:', index)
            new_index.append(index)
            # return new_index

    return new_index

# print(linear_search(3, [1,2,3]))
# print(linear_search(5, [1,2,3,5,5]))